cluster/log-dump/log-dump.sh

```diff
@@ -238,6 +238,33 @@ function save-windows-logs-via-diagnostics-tool() {
     fi
 }
 
+# Saves log files from SSH
+function save-windows-logs-via-ssh() {
+    export-windows-docker-event-log "${node}"
```

Comment:
Does node need to be passed as an arg?

Please do a test run with the updated code if possible to make sure there are no errors :)

Reply:
Same thing for dest_dir

Reply:
Yeah, tested, it works fine. Looks like bash have [dynamic scoping](https://unix.stackexchange.com/questions/282557/scope-of-local-variables-in-shell-functions), the inner function have visibility to local variables of outer function. 
But should pass as an arg for clarity and reliability. Will fix it.

Reply:
Thanks. Bash is the worst.